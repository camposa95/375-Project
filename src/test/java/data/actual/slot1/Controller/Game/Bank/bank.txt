WOOL: 19
ORE: 17
GRAIN: 15
LUMBER: 15
BRICK: 17
