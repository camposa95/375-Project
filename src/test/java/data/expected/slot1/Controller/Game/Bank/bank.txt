WOOL: 19
GRAIN: 15
ORE: 17
LUMBER: 15
BRICK: 17
